pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/AvoidMessageDigestField.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data
+    xmlns="http://pmd.sourceforge.net/rule-tests"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/rule-tests http://pmd.sourceforge.net/rule-tests_1_0_0.xsd">
+    <test-code>
+        <description><![CDATA[
+Basic test case for AvoidStringBufferField
+     ]]></description>
+        <expected-problems>2</expected-problems>
+        <code><![CDATA[
+import java.security.MessageDigest;
+
+public class Foo {
+    private MessageDigest md;
+    private static MessageDigest staticMd;
+}
+     ]]></code>
+    </test-code>
+</test-data>
```

Comment:
We definitively need an example, that verifies, that we don't flag *valid* uses of MessageDigest. We don't want to introduce false-positive...

Reply:
Added